graphblas/io.py

```diff
@@ -220,6 +220,54 @@ def from_awkward(A, *, name=None):
     raise ValueError(f"Invalid format for Matrix: {format}")
 
 
+def from_pydata_sparse(A, *, dup_op=None, name=None):
+    """Create a Vector or a Matrix from a pydata.sparse array or matrix.
+
+    Input data in "gcxs" format will be efficient when importing with SuiteSparse:GraphBLAS.
+
+    Parameters
+    ----------
+    A : sparse
+        PyData sparse array or matrix
+    dup_op : BinaryOp, optional
+        Aggregation function for formats that allow duplicate entries (e.g. coo)
+    name : str, optional
+        Name of resulting Matrix
+
+    Returns
+    -------
+    :class:`~graphblas.Vector`
+    :class:`~graphblas.Matrix`
+    """
+    if A.ndim > 2:
+        raise _GraphblasException("m.ndim must be <= 2")
+
+    if A.ndim == 1:
+        # the .asformat('coo') makes it easier to convert dok/gcxs using a single approach
+        _A = A.asformat("coo")
+        return _Vector.from_coo(
+            _A.coords, _A.data, dtype=_A.dtype, size=_A.shape[0], dup_op=dup_op, name=name
+        )
+
+    # handle two-dimensional arrays
+    import sparse as _sp
```

Comment:
Is `sp` a standard short name for `sparse`? I think I sometimes see `sp` as short for `scipy`. Also, no need for a preceding underscore within the function.